osu.Game.Rulesets.Taiko/Skinning/Legacy/LegacyDrumRoll.cs

```diff
@@ -47,13 +48,13 @@ private void load(ISkinSource skin, OsuColour colours)
                     Anchor = Anchor.CentreRight,
                     Origin = Anchor.CentreLeft,
                     RelativeSizeAxes = Axes.Both,
-                    Texture = skin.GetTexture("taiko-roll-end", WrapMode.ClampToEdge, WrapMode.ClampToEdge),
+                    Texture = skin.GetTexture("taiko-roll-end", WrapMode.ClampToEdge, WrapMode.ClampToEdge)?.WithMaximumSize(new Vector2(128, 256)),
```

Comment:
Why are these 256 based, not 128?

Also adjusting this doesn't affect the test scene at all?

Reply:
Uhh...https://github.com/ppy/osu/pull/24706/commits/9d1753977858dd0e1e0c4312db964749931c11f3

And yes, adjusting doesn't affect because the sprite is resized regardless of the texture, see https://github.com/ppy/osu/pull/24706#discussion_r1316352130.